### PR TITLE
feat(web): switch to react (Partial Implementation)

### PR DESCRIPTION
**Pull request by McPizza0**
## What does this PR do?

Ref #317 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [x] Enhancement (small improvements)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#351** <a href="https://app.graphite.dev/github/pr/un/inbox/351?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#353** <a href="https://app.graphite.dev/github/pr/un/inbox/353?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web-app/components/settings/addTotp.vue [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
   // creation step
   const qrUri = ref('');
-  const qrCode = useQRCode(qrUri, { width: 200, height: 200, margin: 5 });
+  const qrCode = useQRCode(qrUri, { width: 200, margin: 5 });
```
Comment (McPizza0) [inline apps/web-app/components/settings/addTotp.vue:40]: @BlankParticle why was the height removed?
we need it to be square, there was a time when it didnt render properly

File: apps/web-app/components/settings/security/totpReset.vue [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
   // creation step
   const qrUri = ref('');
-  const qrCode = useQRCode(qrUri, { width: 200, height: 200 });
+  const qrCode = useQRCode(qrUri, { width: 200 });
```
Comment (McPizza0) [inline apps/web-app/components/settings/security/totpReset.vue:22]: see other comment about qr code height

File: packages/tiptap/extensions.ts [hunk 3]
```diff
@@ -19,6 +20,10 @@ export const tipTapExtensions: AnyExtension[] = [
       crossorigin: 'use-credentials'
     }
   }),
+  Markdown.configure({
+    html: false,
+    transformCopiedText: true
+  }),
```
Comment (McPizza0) [inline packages/tiptap/extensions.ts:26]: <i class='graphite__hidden'>[Re: lines 10 to 26]<br/><br/></i>

I think we need an export for vue version to ensure its not gonna break<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/un/inbox/353?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by BlankParticle — APPROVED** [review]
@McPizza0 has reviewed the code, but I need to approve as he has made the PR :)

**Pull request merged**